mojap_metadata/converters/glue_converter/__init__.py

```diff
@@ -344,6 +353,60 @@ def generate_from_meta(
         )
         return spec
 
+    def generate_to_meta(self, glue_schema: Union[dict, str]):
+        raise NotImplementedError()
+
+
+class GlueTable(BaseConverter):
+
+    gc = GlueConverter()
+
+    def __init__(self, options: BaseConverterOptions = BaseConverterOptions):
+        super().__init__(options)
```

Comment:
Atm the `gc` object is attached to the class. So you basically have a singleton attached to the GlueTable class which I don't think we want (could be wrong) but I think this is what we want?

```suggestion
    def __init__(self, options: GlueConverterOptions = None):
        super().__init__(None)
        self.gc = GlueConverter(options)
```

Reply:
the options here are for the GlueTable, not the glue converter - although now you point it out, there should be a away to parse options to GlueConverter here, I would suggest:
```suggestion
    def __init__(self, options: BaseConverterOptions = BaseConverterOptions, gc_options = None):
        super().__init__(options)
        self.gc = GlueConverter(gc_options)
```
The alternative is that we don't inherit from `BaseConverter` for `GlueTable` as I just did that for consistency

Reply:
Well I think what we want here is something that either:
- Inherits directly from the `GlueConverter` and then just overload the `generate_from_meta` to do the added stuff. This means you have to rewrite a lot of the functionality in the converter method which doesn't seem ideal.
- Inherits from the BaseConverter method so it just has basic functionality, give it a GlueConverter object and then you only need to overwrite the `generate_from_meta` which calls the `gc.generate_from_meta` under the hood. I don't think you need to give two sets of options (see the other Converters that don't actually take options)